talktorials/5_compound_clustering/T5_compound_clustering.ipynb

```diff
@@ -142,7 +142,7 @@
        "        "
```

Comment:
<p>@rhjohnstone, excellent point - let me see while reviewing if we had a rational to not do this, but I think we just forgot about this option.</p><p> </p><p>As @jaimergp pointed out, let's make this line a bit shorter.</p><p> </p><p>Instead of this</p><p> </p><p><code>clusters = sorted(clusters, key=lambda x: len(x), reverse=True)</code></p><p> </p><p>let's do this</p><p> </p><p><code>clusters = sorted(clusters, key=len, reverse=True)</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/TeachOpenCADD/pull/38/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>